src/commands/moderation/repel.ts

```diff
@@ -131,7 +131,10 @@ const getTextChannels = (interaction: ChatInputCommandInteraction) => {
     return [];
   }
   const channels = getPublicChannels(interaction.guild).map((c) => c);
-  return channels;
+  return [
+    interaction.channel as TextChannel,
+    ...channels.filter((c) => c.id !== interaction.channelId),
```

Comment:
use full variable names. what is c?

```suggestion
    ...channels.filter((channel) => channel.id !== interaction.channelId),
```